spec-main/api-browser-window-spec.ts

```diff
@@ -1240,7 +1238,8 @@ describe('BrowserWindow module', () => {
     })
   })
 
-  describe('BrowserWindow.setOpacity(opacity)', () => {
+
+  ifdescribe(process.platform !== 'linux')('BrowserWindow.setOpacity(opacity)', () => {
```

Comment:
Given that this change also normalizes the result of `getOpacity` on Linux to 1, we should also test for that!